docs/user_guide/capsules/isolation.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: documentation
+title: Capsule Isolation
+sidebar: sidebars/documentation.html
+---
+
+# Capsule Isolation
+
+The goal of Capsule Isolation is to provide a single endpoint for all of a client's communication, so that in remote network segments, you need only open Firewall ports to the Capsule itself.
+
+![Diagram of Communication](./isolation.png)
+
+## Content and Configuration Services
+
+There are 4 primary areas that require client communication:
+
+### Package Installation
```

Comment:
I'd consider using `####` given the above sentence gets lost by the size of the headers of each. You could also drop the colon above since you aren't doing a traditional list here.